libr/core/cmd_print.c

```diff
@@ -2804,6 +2807,31 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 						as->block[p].strings);
 			}
 			break;
+		case 'e':
+			blockptr = malloc (ate - at);
```

Comment:
move the logic of this command into a separate function and just call it frmo here. this way you can reduce the scope of the blockptr variable and make that switch/case more readable